ocfweb/tv/templates/tv/base.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
 
-<html>
+<html class="{{base_css_classes}}">
```

Comment:
Wait...we shouldn't set the background on the `<html>` element, we should be setting it on the `<body>` element.

Reply:
isn't having a class on the `<html>` consistent with every other page?
setting the background on `<body>` doesn't solve the issue with body having padding at the top because of the navbar

Reply:
> isn't having a class on the `<html>` consistent with every other page?

Yes, but we never set the `background` attribute on an `<html>` element. The proper way to add a background for a page is to put it on the `<body>` element.

> setting the background on `<body>` doesn't solve the issue with body having padding at the top because of the navbar

Whether there is padding or not shouldn't affect the positioning of the background, since padding is space inside the element's box. If the body padding is still an issue, you can manually override it to zero.